addons/binding/org.openhab.binding.pulseaudio/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="pulseaudio"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <channel-type id="volume">
+        <item-type>Dimmer</item-type>
+        <label>Volume</label>
+        <description>Volume of an audio item in percent</description>
+        <category>SoundVolume</category>
+        <tags>
```

Comment:
please remove the tags from all your channel-types - we do not yet have defined a central tag library, so we will first have to work this out before bindings should define them.